python/lsst/fgcmcal/fgcmOutputProducts.py

```diff
@@ -341,19 +341,18 @@ def runDataRef(self, butler):
                           "fitCycleConfig.doReferenceCalibration")
 
         # And make sure that the atmosphere was output properly
-        if (self.config.doAtmosphereOutput and
-                not butler.datasetExists('fgcmAtmosphereParameters', fgcmcycle=self.config.cycleNumber)):
-            raise RuntimeError("Atmosphere parameters are missing for cycle %d." %
-                               (self.config.cycleNumber))
+        if (self.config.doAtmosphereOutput
+                and not butler.datasetExists('fgcmAtmosphereParameters', fgcmcycle=self.config.cycleNumber)):
+            raise RuntimeError(f"Atmosphere parameters are missing for cycle {self.config.cycleNumber}.")
 
-        if ((self.config.doReferenceCalibration or self.config.doRefcatOutput) and
-                (not butler.datasetExists('fgcmStandardStars',
+        if ((self.config.doReferenceCalibration or self.config.doRefcatOutput)
+            and (not butler.datasetExists('fgcmStandardStars',
```

Comment:
Or this one? (which is where I would normally put it)

Reply:
I think we get away with this because there is a line below that is indented further.